app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -73,11 +73,23 @@ def flow_session
 
       def analytics_arguments
         {
-          flow_path: flow_path,
+          flow_path: flow_session[:flow_path],
           step: 'verify',
           analytics_id: 'In Person Proofing',
           irs_reproofing: irs_reproofing?,
-        }.merge(**acuant_sdk_ab_test_analytics_args)
+        }.merge(**acuant_sdk_ab_test_analytics_args).
+          merge(**extra_analytics_properties)
+      end
+
+      def extra_analytics_properties
+        extra = {
+          pii_like_keypaths: [[:same_address_as_id], [:state_id, :state_id_jurisdiction]],
+        }
+        unless flow_session[:pii_from_user]&.[](:same_address_as_id).nil?
```

Comment:
Something like this would be easier to read

```suggestion
        unless flow_session.dig(:pii_from_user, :same_address_as_id).nil?
```